ext/AgentsVisualizations/src/spaces/abstract.jl

```diff
@@ -0,0 +1,100 @@
+Agents.agents_space_dimensionality(model::ABM{<:Agents.SpaceType}) = 
```

Comment:
```suggestion
Agents.agents_space_dimensionality(model::ABM) = 
```

Reply:
I think that if you aren't interested in qualifying the space type then you can also remove it

Reply:
That's true. At some point earlier in the process I had `AbstractSpace` here but once I've added the `Nothing` space option, I've changed it to `SpaceType`. I will remove the space type annotations in this file.